### PR TITLE
maxScore access from PaginatorAdapterInterface

### PR DESCRIPTION
**Pull request by istvankis**

### CODE REVIEW

File: Paginator/RawPaginatorAdapter.php [hunk 1]
```diff
@@ -156,6 +162,19 @@ public function getSuggests()
...
+    public function getMaxScore()
+    {
+        if (!isset($this->maxScore)) {
+            $this->maxScore = $this->searchable->search($this->query)->getMaxScore();
+        }
+
+        return $this->maxScore;
+    }
+
+
```
Comment (XWB) [inline Paginator/RawPaginatorAdapter.php]: The second empty line can be removed.

File: Paginator/PaginatorAdapterInterface.php [hunk 2]
```diff
@@ -43,4 +43,10 @@ public function getAggregations();
      * @return mixed
      */
     public function getSuggests();
+
+    /**
+     * Retun max score
+     * @return float
+     */
+    public function getMaxScore();
```
Comment (XWB) [inline Paginator/PaginatorAdapterInterface.php]: Can you add this interface change to the changelog?

**Comment by istvankis** [comment]
can we add this feature to 4.0.x branch?

**Comment by XWB** [comment]
@istvankis As 4.0 has been released already, changing interfaces is not possible anymore.

**Pull request merged**